### ISSUE TITLE
Automatically set-up email-in addresses based on category slug

### ISSUE THREAD

**Issue by LeoMcA** [mozilla/discourse#19]
https://github.com/mozilla/parsys/issues/35

**Comment by LeoMcA** [comment]
Blocked by: https://github.com/mozilla/discourse/issues/18

**Comment by LeoMcA** [comment]
Duplicate of #208
